src/objective-c/tests/BUILD

```diff
@@ -24,13 +24,22 @@ load("@build_bazel_rules_apple//apple:resources.bzl", "apple_resource_bundle")
 load("@build_bazel_rules_apple//apple:ios.bzl", "ios_application", "ios_unit_test")
 load("@build_bazel_rules_apple//apple:macos.bzl", "macos_unit_test")
 load("@build_bazel_rules_apple//apple:tvos.bzl", "tvos_application", "tvos_unit_test")
+load("@build_bazel_rules_apple//apple/testing/default_runner:ios_test_runner.bzl", "ios_test_runner")
 
 licenses(["notice"])
 
 package(default_visibility = ["//visibility:public"])
 
 exports_files(["LICENSE"])
 
+ios_test_runner(
+    name = "ios_test_runner_with_env",
+    test_environment = {
```

Comment:
can we check if using env paramters for ios_unit_test (https://github.com/bazelbuild/rules_apple/blob/master/doc/rules-ios.md#ios_unit_test-env) would work ? that way we don't have to define extra runner for each test.

Reply:
looks like we may also pass this in via test_env (https://bazel.build/reference/command-line-reference#flag--test_env)